packages/rocket-batch-file-process-aws-infrastructure/README.md

```diff
@@ -32,9 +38,16 @@ Booster.configure('development', (config: BoosterConfig): void => {
   config.provider = Provider([{
     packageName: '@boostercloud/rocket-batch-file-process-aws-infrastructure', 
     parameters: {
-      bucketName: 'test-bucket-name', 
-      chunkSize: '2',
-    }
+      config: {
+        bucketName: 'test-bucket',
+        chunkSize: '2',
+      },
+      rowEvent: {
+        entityId: 'id',
```

Comment:
Aren't we restricting the user by adding `entityId`? I mean, we'd be supposing that the user wants to update the exact same entity all the time, no matter the S3 structure.

Reply:
I didn't get this comment @juanjoman We are passing the entityId of the event we defined in our booster app. The rocket needs to know about which attribute is the entity ID because when saving a booster event we need this piece of info https://github.com/boostercloud/booster/blob/606d13c4e0bee83b777c631f74a7c1e571724780/packages/rocket-batch-file-process-aws-infrastructure/src/file-to-line-event-lambda/index.ts#L74